lib/data_sources/repo_docs.rb

```diff
@@ -136,19 +136,24 @@ def sync_repository
   def versions
     branches = git_branches
     all = git_tags
-      .select { |v| v.match(VERSION_REGEXP) }
-      .map { |v| v.delete_prefix('v').split('.')[0, 2].join('.') }
-      .uniq
-      .select { |v| branches.include?('release-' + v) }
-      .sort_by { |v| v.split('.').map(&:to_i) }
-      .reverse
-
-    # Number of versions is reduced to speed up site compilation time.
+            .select { |v| v.match(VERSION_REGEXP) }
+            .map { |v| v.delete_prefix('v').split('.')[0, 2].join('.') }
+            .uniq
+            .select { |v| branches.include?('release-' + v) }
+            .sort_by { |v| v.split('.').map(&:to_i) }
+            .reverse
```

Comment:
AFAICS nothing changed in these lines, maybe keep the same indent of 2 as before and everywhere else?